step/step.go

```diff
@@ -84,6 +84,8 @@ func Cmd(name string, args ...string) Step {
 		output = string(out)
 		if exit, ok := err.(*exec.ExitError); ok {
 			err = fmt.Errorf("%s:\n%s", err.Error(), string(exit.Stderr))
+		} else if err != nil {
+			err = fmt.Errorf("%s", err.Error())
```

Comment:
What does this achieve? `.Error()` should be printed by default, no?

Reply:
Not 100% sure ... just copying from above to handle the case where the error is not an exit error.

Would be great to figure out how to test this kind of machinery better.

Reply:
I think the case above just tries to concatenate the error with stderr. Without stderr, we should be able to just return `err`.

Reply:
Oh yeah you're right 

I'm finding this code hard to follow as it seems like it always actually returns an empty string ... but the actual useful return is done by setting the `output` variable ... which is set in the `.Return(&output)`. Guess this isn't the source of the lost console output then 😞